apps/devhub/tasks.py

```diff
@@ -67,16 +68,29 @@ def validate_and_submit(addon, file_, listed=None):
 def submit_file(addon_pk, file_pk):
     addon = Addon.unfiltered.get(pk=addon_pk)
```

Comment:
We have [Addon.no_cache()](https://github.com/washort/django-cache-machine/blob/master/caching/base.py#L61) if that's what you're talking about?